lib/src/services/databases/calendar/implementation.dart

```diff
@@ -34,12 +34,13 @@ class CloudCalendar implements CalendarInterface {
 	Future<List<Map>> getSchedules() async => List<Map>.from(
 		(await schedules.throwIfNull("Cannot find schedules")) ["schedules"]
 	);
-
+	@override
+	Future<Map<String, String>> getDefaultSchedules() async => Map.from(
```

Comment:
```suggestion
	Future<Map<String, String>> getDefaultSchedules() async => 
```

You don't need `Map` here, unless you need to cast to a `Map<String, String>` specifically (which you might).